### PR TITLE
CA-186157: rrdp_iostat: Use new tapdisk3 stats struct

### PR DESCRIPTION
**Pull request by rafalmiel**

### CODE REVIEW

**Comment by xen-git** [comment]
Can one of the admins verify this patch for testing?

File: rrdp_iostat.ml [hunk 1]
```diff
@@ -343,12 +339,27 @@ module Blktap3_stats_wrapper = struct
...
+					match read_raw_blktap3_stats tdpid domid devid with
+					| Some stat -> ((domid, devid), stat) :: acc
+					| None -> acc
+					)
+			with e ->
+				D.error "Error while looking up tapdisk pid: %s" (Printexc.to_string e);
+				acc
+		) [] shm_vbds
+
+		(* List.fold_left (fun acc vbd ->
```
Comment (johnelse) [inline rrdp_iostat.ml]: Please delete this code rather than commenting it out.

**Comment by johnelse** [comment]
Other than the commented code, I think this can go in once the blktap header changes are in trunk-ring3. Thanks!

**Comment by rafalmiel** [comment]
Yes, there is already a pull request, waiting for the review

**Comment by johnelse** [comment]
Looks good, thanks!

**Pull request merged**